libs/datasets/combined_datasets.py

```diff
@@ -231,6 +231,13 @@ def load_us_timeseries_dataset(
     return MultiRegionDataset.read_from_pointer(pointer)
 
 
+@functools.lru_cache(None)
+def load_test_dataset() -> MultiRegionDataset:
+    wide_dates_path = dataset_utils.REPO_ROOT / "tests/data/test-combined-wide-dates.csv"
```

Comment:
Could this path be in a constant somewhere and shared with update_test_combined_data() ?

Reply:
Done